flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -1130,259 +1134,263 @@ object AggregateUtil {
     // create aggregate function instances by function type and aggregate field data type.
     aggregateCalls.zipWithIndex.foreach { case (aggregateCall, index) =>
       val argList: util.List[Integer] = aggregateCall.getArgList
-      if (argList.isEmpty) {
-        if (aggregateCall.getAggregation.isInstanceOf[SqlCountAggFunction]) {
-          aggFieldIndexes(index) = Array[Int](0)
```

Comment:
Couldn't just change this line to `aggFieldIndexes(index) = Array[Int](-1)` instead of touching each line of this method?

Reply:
I think it should work, but I do not want deal with `SqlCountAggFunction` twice, so i remove the `SqlCountAggFunction` from case condition.